test/core/api/state/state_api_service_test.cpp

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include "api/state/state_jrpc_processor.hpp"
+#include "api/state/state_api_service.hpp"
+
+using kagome::api::StateJrpcProcessor;
+using kagome::api::StateApiService;
+
+
+
+TEST(StateJrpcProcessorTest, Create) {
+  StateApiService service;
```

Comment:
Best test ever) Is it finished or you forgot to remove it?

Reply:
Definitely forgot. I realised that it's not needed but forgot to remove it.